volttrontesting/services/historian/test_multiplatform.py

```diff
@@ -57,7 +57,13 @@
 from volttrontesting.utils.platformwrapper import PlatformWrapper
 from volttrontesting.fixtures.volttron_platform_fixtures import get_rand_vip, \
     get_rand_ip_and_port
-from volttron.utils.rmq_setup import start_rabbit, stop_rabbit
+
+from volttron.platform import is_rabbitmq_available
+if is_rabbitmq_available():
+    from volttron.utils.rmq_setup import start_rabbit, stop_rabbit
+else:
+    pytest.skip("Pika is not installed", allow_module_level=True)
```

Comment:
Again, only the rmq test cases should be skipped. you can set a variable inside if and use that to skip the rmq test case alone